kitsune/sumo/static/sumo/js/rickshaw_utils.js

```diff
@@ -308,8 +308,8 @@
     }
 
     // Rickshaw gets angry when its data isn't sorted.
-    function sortCallback (a, b) {
-      return a.x - b.x;
+    function sortCallback (v1, v2) {
```

Comment:
Generally, it is fine with them. But [http://eslint.org/docs/rules/no-shadow](http://eslint.org/docs/rules/no-shadow) is on by default, and there is a variable in the parent scope named `b`. There are a few changes like this that are kind of awkward.